README.md

```diff
@@ -30,7 +30,7 @@ How to install and run Brackets
 
 Installers for the latest stable build for Mac, Windows and Linux (Debian/Ubuntu) can be [downloaded here](http://brackets.io/).
 
-The Linux version has most of the features of the Mac and Windows versions, but
+The Linux version has more features than the Mac and Windows versions, but
```

Comment:
This sentence is trying to say "Most, but not all, of the features implemented for Mac and Windows versions have been implemented for Linux". It is *not* true the "Linux has more features".